sh/tests/easy-perm_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+chmod 303 easy-perm/*
+
+challenge() {
+    submitted=$(bash "$script_dirS"/student/easy-perm.sh && ls -l "$1")
```

Comment:
you have to check the permission only
with ls -l you will compare group of files infos, which will impact the tests results.

Reply:
the exercise also demands that the file struct is the exact same one as ours so i used ls -l because of that should we change it?

Reply:
fixed by MS
MS in discord: right, i will use awk to filter in the test, Thanks 👍